packages/perseus/src/hints-renderer.jsx

```diff
@@ -72,11 +74,18 @@ class HintsRenderer extends React.Component<Props, State> {
             this._cacheHintImages();
         }
 
+        // Force number to make Flow happy
+        const prevHintsVisible = prevProps.hintsVisible || 0;
+        const nextHintsVisible = this.props.hintsVisible || 0;
```

Comment:
Question: Is the "||" here in case the left hand side is null? If so, would the nullish coalescing operator work better? Or what else could the lefthand side be?

Reply:
`hintsVisible` is an optional prop, so it can be `undefined`. I need it to be a number, so if it is `undefined` I instead use `0`.